README.md

```diff
@@ -103,6 +103,18 @@ import validate, { Joi } from ('koa-router-joi-validation');
 
   **default** `false`
 
+#
+
+- `alternate`
```

Comment:
@lloria91 if we could put the link of the Joi alternative function, to explain that this is a wrapper of that function